__init__.py

```diff
@@ -16,7 +16,9 @@
 
 load_dotenv()
 
-BASE_COMFY_DIR: str = os.path.dirname(os.path.realpath(__file__)).split("custom_nodes")[0]
+BASE_COMFY_DIR: str = os.path.dirname(os.path.realpath(__file__)).split("custom_nodes")[
+    0
```

Comment:
Did you format that with ruff? I just saved with my VS code and ruff plugin, and it re-formatted this line. Also several lines below, that only contain format changes where restored to the previous formatting.

Reply:
I believe I did, it's my default formatter